### ISSUE TITLE
During the training process, the loss has been rising

### ISSUE THREAD

**Issue by amnesiac555** [VITA-Group/DeblurGANv2#84]
![Snipaste_2020-10-04_10-52-50](https://user-images.githubusercontent.com/42953443/95005867-ec24db00-062f-11eb-8870-474122f24dca.png)
i use the mobilenet and fpn_inception, training data is Gopro, config is as follows:
anyone has trained successfully？

project: deblur_gan
experiment_desc: fpn

train:
  files_a: ./data/GOPRO/GOPRO_3840FPS_AVG_3-21/train/blur/**/*.png
  files_b: ./data/GOPRO/GOPRO_3840FPS_AVG_3-21/train/sharp/**/*.png
  size: &SIZE 256
  crop: random
  preload: &PRELOAD false
  preload_size: &PRELOAD_SIZE 0
#  bounds: [0, .9]
  scope: geometric
  corrupt: &CORRUPT
    - name: cutout
      prob: 0.5
      num_holes: 3
      max_h_size: 25
      max_w_size: 25
    - name: jpeg
      quality_lower: 70
      quality_upper: 90
    - name: motion_blur
    - name: median_blur
    - name: gamma
    - name: rgb_shift
    - name: hsv_shift
    - name: sharpen

val:
  files_a: ./data/GOPRO/GOPRO_3840FPS_AVG_3-21/test/blur/**/*.png
  files_b: ./data/GOPRO/GOPRO_3840FPS_AVG_3-21/test/sharp/**/*.png
  size: *SIZE
  scope: geometric
  crop: center
  preload: *PRELOAD
  preload_size: *PRELOAD_SIZE
#  bounds: [.9, 1]
  corrupt: *CORRUPT

phase: train
warmup_num: 3
model:
  g_name: fpn_mobilenet
  blocks: 9
  d_name: double_gan # may be no_gan, patch_gan, double_gan, multi_scale
  d_layers: 3
  content_loss: perceptual
  adv_lambda: 0.001
  disc_loss: wgan-gp
  learn_residual: True
  norm_layer: instance
  dropout: True

num_epochs: 200
#train_batches_per_epoch: 1000
#val_batches_per_epoch: 100
batch_size: 1
image_size: [256, 256]

optimizer:
  name: adam
  lr: 0.0001
scheduler:
  name: linear
  start_epoch: 50
  min_lr: 0.0000001

**Comment by amnesiac555** [comment]
i didn't change training code, feel confused about it

**Comment by NoviceCrom** [comment]
Maybe you can change the "disc_loss" in the config.yml to ragan_ls which is stated in the paper.

**Comment by amnesiac555** [comment]
> Maybe you can change the "disc_loss" in the config.yml to ragan_ls which is stated in the paper.

OK   THANKS ！  i will try it immediately

**Comment by ldlshizhu** [comment]
请问你训练成功了吗

**Comment by ldlshizhu** [comment]
你好，我看到你train_batches_per_epoch: 1000
val_batches_per_epoch: 100这两行注释掉了，但是我下载的代码里没有注释这两行。请问是什么原因呢

**Comment by ldlshizhu** [comment]
> ragan_ls

你好，我没太懂怎么改ragan_ls?

**Comment by zmwang0574** [comment]
> 你好，我看到你train_batches_per_epoch: 1000
> val_batches_per_epoch: 100这两行注释掉了，但是我下载的代码里没有注释这两行。请问是什么原因呢

Hi, I'm also wondering why you # these two lines. Are they unnecessary for the training? BTW, I'm also wondering what does "bounds" do, which is also #-ed by you.

**Comment by cianhwang** [comment]
> > 你好，我看到你train_batches_per_epoch: 1000
> > val_batches_per_epoch: 100这两行注释掉了，但是我下载的代码里没有注释这两行。请问是什么原因呢
> 
> Hi, I'm also wondering why you # these two lines. Are they unnecessary for the training? BTW, I'm also wondering what does "bounds" do, which is also #-ed by you.

Commenting train/val_batches_per_epoch ensures each epoch the network sees all the training/validation samples. "bounds" s are intended to split the train/val set, which should all be set to [0.0, 1.0] if you specify separate training/validation sets.

**Comment by KMCzajkowski** [comment]
Have you solved this issue? I also have a similar problem. The loss is not improving during training regardless of what I do.
